mocks/mock-utils.js

```diff
@@ -0,0 +1,41 @@
+const getEmptyApiResponse = () => {
+  return {
+    meta: {
+      next: null,
+    },
+    items: [],
+  };
+};
+
+const getBaseTimeEntry = () => {
+  return {
+    id: '',
+    tenantId: '00000000-0000-0000-0000-000000000000',
+    version: 0,
+    ownerId: 1,
+    timePeriodTypeId: '',
+    shiftType: '',
+    actualStartTime: '',
+    actualEndTime: '',
+    plannedStartTime: null,
+    plannedEndTime: null,
+    createdAt: '2022-08-24T12:01:43.786+00:00',
+    updatedAt: '2022-08-24T12:01:43.786+00:00',
+    deleted: false,
```

Comment:
The following fields are being used and you can remove the other fields. The other fields are required when we start implementing Accrual
id;
tenantId;
ownerId;
timePeriodTypeId;
timePeriodType;
shiftType;
actualStartTime;
actualEndTime;
createdAt;
updatedAt;